vaxrank/cli.py

```diff
@@ -180,6 +182,11 @@ def add_output_args(arg_parser):
         type=int,
         help="Number of mutations to report")
 
+    output_args_group.add_argument(
+        "--output-passing-variants-csv",
```

Comment:
Worth explaining that "passing" here refers to the variant caller, not some kind of filtering done by Vaxrank.